### ISSUE TITLE
Support for labels/tags for individual default metrics.

### ISSUE THREAD

**Issue by surbhig213** [yabeda-rb/yabeda#22]
Hey @Envek, 
How can I add a custom tag to default sidekiq metrics. Right now a default tag gets added to all the sidekiq metrics. Is there a way to add a tag to just one metrics?

**Comment by Envek** [comment]
No, it is not possible now. It should be handled on main `yabeda` gem level, so I transferred this issue there.

Can you please tell more about your use case? Why you need to add label only to one metric, or why you don't want not to add it to others? Usually extra labels doesn't hurt and you can ignore them

**Comment by surbhig213** [comment]
Hey @Envek ,
The use case is same as this issue that I raised.
https://github.com/yabeda-rb/yabeda/issues/21
Our system expects metrics with a particular label, so I was trying to add that label to a few metrics only that we need.
